### ISSUE TITLE
iperf3

### ISSUE THREAD

**Issue by pjebs** [yeahdongcn/iperf-iOS#2]
I looked at your code and it worked well.

I was looking at your project and trying to figure out how to add iperf3 client to my own iOS project.

If I want to use the latest version of iperf3, do I just have to recopy all the files from here:

1) Copy files from here to Xcode project: https://github.com/esnet/iperf/tree/master/src
2) Delete out Makefile.am and Makefile.in
3) Rename .in files by removing .in extension

Is that correct?
Do I need to make other configurations and build setting changes?

**Comment by yeahdongcn** [comment]
Right, since my PR is accepted, you can use the latest source from https://github.com/esnet/iperf.git, import all source files inside the src folder and make sure they are under the Project -> Build Phases -> Compile Sources.

**Comment by pjebs** [comment]
Do you know of any resources that you may have used to learn the C code.
The documentation is mainly how to configure the client and server and the various command line options.

Right now, I'm finding your code hard to understand. I'm currently just using it as is.

**Comment by yeahdongcn** [comment]
You may take a look at the two examples (https://github.com/esnet/iperf/blob/master/examples/mic.c and https://github.com/esnet/iperf/blob/master/examples/mis.c) which demonstrate how to use the source.

**Comment by pjebs** [comment]
Someone added authentication as an extra command line flag to iperf3:

https://github.com/esnet/iperf/pull/242/files

In your code, how would I set the `-a` flag with a string indicating the authentication token set at the server?

**Comment by yeahdongcn** [comment]
You could samplely add this line `iperf_set_test_authentication(test, "authentication");`

**Comment by yeahdongcn** [comment]
I just updated the demo project, you could take a look
